image/git-init/git/git.go

```diff
@@ -51,7 +51,7 @@ func run(logger *zap.SugaredLogger, dir string, args ...string) (string, error)
 	}
 	if err := c.Run(); err != nil {
 		logger.Errorf("Error running git %v: %v\n%v", args, err, output.String())
-		return "", err
+		return output.String(), err
```

Comment:
Why do we return the output if we already logged it ? 🤔

Reply:
fixed. Code of remote origin setting follows read-check-set pattern.